bin/dipy_median_ostu

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env python
+
+from dipy.workflows.segment import median_otsu_bet
+from dipy.fixes import argparse as arg
+
+parser = arg.ArgumentParser()
+parser.add_argument('input', action='store')
+parser.add_argument('--out_dir', type=str, default='')
+parser.add_argument('--save_masked', action='store_true')
+parser.set_defaults(save_masked=False)
+parser.add_argument('--radius', type=int, default=4)
```

Comment:
Hmm... think of this for a bit as a client/server problem. Let's say we had a client that would get some inputs and send them to the server to run the workflow.  The client is only able to communicate with the server by sending the inputs as strings. So, I think that the most generic design which we can have for the workflows would be that all parameters are strings. But hey I may be wrong. Happy to hear other opinions about this.